tests/framework/matcher/created.go

```diff
@@ -0,0 +1,42 @@
+package matcher
+
+import (
+	"fmt"
+
+	"github.com/onsi/gomega/types"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"kubevirt.io/kubevirt/tests/framework/matcher/helper"
+)
+
+func BeCreated() types.GomegaMatcher {
```

Comment:
The matcher as such is great but `BeCreated` may be too close to `Exists` matchers, which would assert if the object passed to `Expect` exists.

I did not check how often this matcher is needed. If often, let's just come up with a nicer name.

If less often, it may make sense to create a general-purpose `unstructured` matcher which could be called like this:


```
Expect(ThisVM(vm)).To(MatchUnstructured("status", "created", true))
```

or something along that line.